test/parallel/test-repl.js

```diff
@@ -358,7 +358,7 @@ const errorTests = [
     send: ')',
     expect: 'undefined'
   },
-  // npm prompt error message
+  // `npm` prompt error message.
```

Comment:
I kind of chafe at markdown in comments, so this is (from my perspective) mildly unfortunate. Not blocking--only commenting in case you have another idea. (Maybe we can just add `|npm` to the end of the `ignorePatterns` regexp? Maybe long-term goal can be that the whole regexp will just be a set of words separated by pipe characters like `var|let|const|npm`?

Reply:
We could add a couple of words to the blacklist but I guess it might become relatively big if we only use a blacklist. So I would definitely keep the current ignore pattern as it overall worked very well.

Reply:
Hm, I personally would say this specific case actually benefits by making clear that `npm` is something specific. If it would be a more generic word, it would be difficult to distinguish it from the rest of the comment.